src/Network/Http.php

```diff
@@ -347,6 +357,9 @@ public function send()
         curl_close($curl);
 
         if ($this->streamFile) {
+            rewind($headerStream);
```

Comment:
Are there any concerns here with the temp stream being modified in between `curl_exec` being called and this code being run? Should we namespace the stream at all? Is that even possible?

Reply:
@LukeTowers according to the [PHP docs](https://www.php.net/manual/en/wrappers.php.php), temp streams are not reusable, so it would appear to me that each stream operates on its own. There's no namespaces for it either, which if the previous point is true would make them redundant anyway.

Reply:
Tested it with the following and it seems to be accurate so sounds good to me:

```php
$stream1 = fopen('php://temp', 'r+');
$stream2 = fopen('php://temp', 'r+');

fwrite($stream1, 'test');

rewind($stream2);
stream_get_contents($stream2);
```